types/xml-formatter/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for xml-formatter 1.1
+// Project: https://github.com/chrisbottin/xml-formatter/ (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Joachim Holwech <https://github.com/holwech>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ Note that ES6 modules cannot directly export callable functions.
```

Comment:
These comments look like they might be from a template.  Consider updating them with content specific to xml-formatter.